tests/test_frame.py

```diff
@@ -728,17 +728,15 @@ def gethead(s: pd.Series, y: int) -> pd.Series:
     )
 
 
-def test_types_applymap() -> None:
-    with pytest_warns_bounded(
-        FutureWarning, "DataFrame.applymap has been deprecated", lower="2.0.99"
-    ):
-        df = pd.DataFrame(data={"col1": [2, 1], "col2": [3, 4]})
-        df.applymap(lambda x: x**2)
-        df.applymap(np.exp)
-        df.applymap(str)
-        # na_action parameter was added in 1.2.0 https://pandas.pydata.org/docs/whatsnew/v1.2.0.html
-        df.applymap(np.exp, na_action="ignore")
-        df.applymap(str, na_action=None)
+# In pandas 2.1.0 and later: applymap is renamed map, applymap is deprecated
+def test_types_map() -> None:
```

Comment:
Not all tests reference the relevant github issue, but I believe (@Dr-Irv) we try to add # GHXXX as a comment on the next line

Reply:
not sure what you'd like here

Reply:
```py

def test_types_map() -> None:
    # GH774
    ...
```

Reply:
updated

Reply:
updated